CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+## Contributing
+
+Slng-node is an open source project and we're always looking to make our product better. Feel free to browse our [issues](https://github.com/5-gwoap/slng-node/issues) for a fix/enhancement you may be interested in helping out with.
+
+1. Fork this project
+2. Choose an issue
+3. Fix it!
+4. Send a PR to `development`
+
+#### Branch Names
+Try to keep branch names following the following format:
+
+  `slng-<issue number\>-topic`
```

Comment:
Should this be `<issue number>-topic` and not have the extra `\`

Reply:
Markdown was processing the <> and not rendering the text correctly w/o escaping the last >.